pages/discover.js

```diff
@@ -251,6 +252,7 @@ const DiscoverPage = ({ router, intl }) => {
                           setRouteParam('sort', selected && selected.key);
                         }
                       }}
+                      minWidth={space}
```

Comment:
What is this supposed to do?

Reply:
@Betree I don't know if you could give me a head start (steps) to take in resolving that, because i would love to contribute the project

Reply:
@Nelson-Chinedu I'll be happy to help you if you have any specific questions, I tried to specify what's still needed in https://github.com/opencollective/opencollective-frontend/pull/2348#pullrequestreview-276382155. What part of it do you need steps for?

Reply:
Ok from the gif image you sent i don't seem to find the sort by when I visit the url what I see is the image below 
![snip](https://user-images.githubusercontent.com/26670804/64033779-1c549d00-cb45-11e9-8ffe-30e8166afb64.PNG)

Reply:
@Nelson-Chinedu This is the old `DeprecatedStyledSelect`, you can tell because the caret on the right is filled, while it's just a line on the new `StyledSelect`. You're probably testing from a wrong URL of from the wrong branch of you local repository.

There's a pull request preview on this PR, you can see the current version on https://opencollective-frontend-ozouacefph.now.sh/discover